api/HTMLElement.json

```diff
@@ -635,10 +635,10 @@
               "version_removed": "79"
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": "8"
```

Comment:
The original version number came from the wiki migration, and is inconsistent with the data of the [`<menu>` HTML element data](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu).  Since a context menu needs a `<menu>` element to be defined, it doesn't make sense that this property is supported before this HTML element.

Reply:
OK, thanks for confirming!